app/services/auth.js

```diff
@@ -83,6 +83,29 @@ export default Service.extend({
 
   redirectUrl: null,
 
+  init() {
+    this._super(...arguments);
+    window.addEventListener('focus', this.checkAuth.bind(this));
```

Comment:
This `bind` signature looks a little bit awkward to me :) I'd suggest an arrow function instead:
```suggestion
    window.addEventListener('focus', () => this.checkAuth());
```
Just an optional suggestion though.

Reply:
You are right. Arrow functions look a way more trendy :-)